packages/orchestrator/lib/client.ts

```diff
@@ -163,4 +176,16 @@ export class OrchestratorClient {
         };
         return this.execute(schedulingProps);
     }
+
+    public async executePostConnection(props: TExecutePostConnectionProps): Promise<TExecuteReturn> {
```

Comment:
you are gonna need to add a TaskPostConnection type and validation in orchestrator and add logic in jobs processor handler so the task can be processed
- https://github.com/NangoHQ/nango/blob/7774be4157ecd235fe737c6c662f368e309b121c/packages/orchestrator/lib/clients/types.ts#L35
- https://github.com/NangoHQ/nango/blob/7774be4157ecd235fe737c6c662f368e309b121c/packages/orchestrator/lib/clients/validate.ts#L24
- https://github.com/NangoHQ/nango/blob/7774be4157ecd235fe737c6c662f368e309b121c/packages/jobs/lib/processor/handler.ts#L6

You don't have to do it in this PR though.

Reply:
Thanks, updated